schema.graphql

```diff
@@ -995,6 +996,7 @@ input AddLayerGroupInput {
   index: Int
   linkedDatasetSchemaID: ID
   name: String
+  markerTitleFieldId: DatasetSchemaFieldID
```

Comment:
The layer to be added is not necessarily a marker, so this name is too specific; how about representativeFieldId?

Reply:
Resolved